foundations/javascript_basics/installing_nodejs.md

```diff
@@ -138,4 +138,5 @@ To run the Node console, open up your terminal and type `node`. Type `.exit` to
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+- Wondering why are we installing Node? you will learn the different uses of Node in the next chapters but if you're curious,
```

Comment:
The learner shouldn't be wondering why we're installing Node. It's answered in the first two sentences of this lesson. However, I do like the video. Why don't we say something like "If you are wanting to learn just a little bit more about NodeJS, check out this short video by Programming with Mosh."

Reply:
I updated the .md file does this satisfy your recommendation sir ?